libs/core/kiln_ai/adapters/chunkers/base_chunker.py

```
@@ -0,0 +1,37 @@
+import logging
+from abc import ABC, abstractmethod
+
+from pydantic import BaseModel, Field
+
+from kiln_ai.datamodel.chunk import ChunkerConfig
+
+logger = logging.getLogger(__name__)
+
+
+class Chunk(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)